src/components/FrontPageButton.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import style from './buttons.scss';
+import Teacher from './TeacherPage';
+
+export default class FrontPageButton extends React.Component {
```

Comment:
Ser at du har brukt extends React.Component istedet for React.createClass. Det er mest bare en syntax-forskjell, men jeg synes vi skal være konsekvente, så enten alle med extends eller alle med createClass. Jeg har ingen sterke meninger ene eller andre veien.

En ting å være klar over, er at ved extends, så kan man ikke bruke mixins. Og dessuten, så må man manuelt binde "this" når man kaller interne funksjoner. Se f.eks. https://toddmotto.com/react-create-class-versus-component/

Jeg foreslår for nå at vi holder oss til React.createClass, men kom gjerne med overbevisende argumenter for at vi skal bruke extends React.Component istedet.